swap/swap_test.go

```diff
@@ -1083,3 +1084,29 @@ func (d *dummyMsgRW) ReadMsg() (p2p.Msg, error) {
 func (d *dummyMsgRW) WriteMsg(msg p2p.Msg) error {
 	return nil
 }
+
+func TestFileLogger(t *testing.T) {
```

Comment:
This test gives me a `no such file or directory` error

Reply:
Mock test as above